core/src/main/java/org/elasticsearch/indices/recovery/RecoveryTargetHandler.java

```diff
@@ -39,11 +39,12 @@
     void prepareForTranslogOperations(int totalTranslogOps, long maxUnsafeAutoIdTimestamp) throws IOException;
 
     /**
-     * The finalize request clears unreferenced translog files, refreshes the engine now that
-     * new segments are available, and enables garbage collection of
-     * tombstone files.
-     **/
-    void finalizeRecovery();
+     * The finalize request clears unreferenced translog files, refreshes the engine now that new segments are available, enables garbage
```

Comment:
You're right, it was already not correct. I pushed 1c71393b7f40a5c3b27d6be1bdbfe9bee2dadff1.